data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -1,530 +1,625 @@
-            side=1
-            x,y=8-30,2-10
-        [/remove_shroud]
-        [remove_shroud]
-            side=1
-            x,y=31-37,4-11
-        [/remove_shroud]
-        [remove_shroud]
-            side=1
-            x,y=37-38,4-5
-        [/remove_shroud]
-        [modify_side]
-            side=3
-            share_vision=all
-        [/modify_side]
-
-        [message]
-            speaker=unit
-            message= _ "Who goes there?"
-        [/message]
-
-        [message]
-            speaker=Gweddry
-            message= _ "We are soldiers of the King of Wesnoth. Will you help us fight these trolls?"
-        [/message]
-
-        [message]
-            speaker=unit
-            message= _ "Aye, we’ll help ye, for we have nae love for the trolls. Anyone attacked by them deserves some help, I think."
-        [/message]
-
-        [if]
-            {VARIABLE_CONDITIONAL undead boolean_equals yes}
-            [then]
-                [message]
-                    speaker=Dacyn
-                    # po: shown only if the undead have arrived
-                    message= _ "Thank you. Now, I invite you to join us. Undead have followed us into these caves, and they will kill you if you stay behind."
-                [/message]
-            [/then]
-            [else]
-                [message]
-                    speaker=Dacyn
-                    # po: shown if the undead didn't show up yet
-                    message= _ "Thank you. Now, I invite you to join us. Undead are in these lands, and they will find these caves."
-                [/message]
-            [/else]
-        [/if]
-
-        [message]
-            speaker=unit
-            message= _ "My clan ha’ lived in these caves for centuries. We’ll no’ be scattered now by a few undead."
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "Very well. The best of luck in your battle. Gweddry, we may be able to avoid the bulk of the trolls by going through the dwarves’ tunnel."
-        [/message]
-    [/event]
-
-    {HOLY_AMULET 12 2}
-
-    # opening the door to the treasure room
-    [event]
-        name=moveto
-        [filter]
-            x=30,32
-            y=9,10
-        [/filter]
-
-        [sound]
-            name=club.ogg
-        [/sound]
-        [terrain]
-            x=31
-            y=10
-            terrain=^Pw/o
-            layer=overlay
-        [/terrain]
-    [/event]
-
-    [event]
-        name=moveto
-        [filter]
-            x,y=35,8
-            side=1
-        [/filter]
-
-        [sound]
-            name=gold.ogg
-        [/sound]
-
-        [message]
-            speaker=unit
-            message= _ "It seems these trolls were hiding some gold. I think I can count fifty gold pieces!"
-        [/message]
+            controller=ai
+            recruit=Troll Whelp
+            gold,income=30,3 # scaling comes from the {GUARDIAN} units, which are worth 13 + 13 + 25 gold
```

Comment:
having another source of scaling doesn't mean that the scaling here should be removed, too, though. The more scaling the better.

Reply:
I'm open to reducing the trolls' gold on easy/normal if you feel that's necessary.  Please do playtest it first though.